packages/cli/src/api/compile.ts

```diff
@@ -118,34 +113,40 @@ function buildExportStatement(expression, namespace: CompiledCatalogNamespace) {
  * Compile string message into AST tree. Message format is parsed/compiled into
  * JS arrays, which are handled in client.
  */
-export function compile(message: string) {
+export function compile(message: string, shouldPseudolocalize: boolean = false) {
   let tokens
 
   try {
     tokens = parse(message)
   } catch (e) {
     throw new Error(
-      `Can't parse message. Please check correct syntax: "${message}" \n \n Messageformat-parser trace: ${e.message}`,
+      `Can't parse message. Please check correct syntax: "${message}" \n \n Messageformat-parser trace: ${e.message}`
     )
   }
-  const ast = processTokens(tokens)
+  const ast = processTokens(tokens, shouldPseudolocalize)
 
   if (isString(ast)) return t.stringLiteral(ast)
 
   return ast
 }
 
-function processTokens(tokens) {
+function processTokens(tokens, shouldPseudolocalize: boolean) {
   // Shortcut - if the message doesn't include any formatting,
   // simply join all string chunks into one message
   if (!tokens.filter((token) => !isString(token)).length) {
```

Comment:
Not your code, but it can be optimized (so it stops iterating once the first non-string item was encountered).
```suggestion
  if (tokens.some((token) => !isString(token))) {
```